### PR TITLE
[WIP] Remove most python2 packages

### PR DESCRIPTION
**Pull request by gytis-ivaskevicius**
###### Motivation for this change
TL;DR: Just playing around, commented out all python2 packages that do not cause any issues.

I have no idea if everything correctly works. I'm just playing around

**EDIT:** Calm down with the comments, I just wanted to eval everything :D 

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by mweinelt** [comment]
@ofborg build nixops

File: pkgs/top-level/python2-packages.nix [hunk 1]
```diff
@@ -6,666 +6,672 @@ self: super:
...
-
-  hypothesis = callPackage ../development/python-modules/hypothesis/2.nix { };
-
-  idna = callPackage ../development/python-modules/idna/2.nix { };
-
-  imagecodecs-lite = disabled super.imagecodecs-lite;
-
-  imbalanced-learn = callPackage ../development/python-modules/imbalanced-learn/0.4.nix { };
-
-  importlib-metadata = callPackage ../development/python-modules/importlib-metadata/2.nix { };
```
Comment (mweinelt) [inline pkgs/top-level/python2-packages.nix:185]: breaks nixops

**Comment by siraben** [comment]
```
error: undefined variable 'rpkg'

       at /home/siraben/.cache/nixpkgs-review/pr-131585/nixpkgs/pkgs/development/tools/fedpkg/default.nix:29:29:

           28|   patches = [ ./fix-paths.patch ];
           29|   propagatedBuildInputs = [ rpkg offtrac urlgrabber fedora_cert ];
             |                             ^
           30|

       … while evaluating 'chooseDevOutputs'

       at /home/siraben/.cache/nixpkgs-review/pr-131585/nixpkgs/lib/attrsets.nix:497:22:

          496|   /* Pick the outputs of packages to place in buildInputs */
          497|   chooseDevOutputs = drvs: builtins.map getDev drvs;
             |                      ^
          498|

       … from call site

       … while evaluating the attribute 'propagatedBuildInputs' of the derivation 'fedpkg-1.29'

       at /home/siraben/.cache/nixpkgs-review/pr-131585/nixpkgs/pkgs/stdenv/generic/make-derivation.nix:203:11:

          202|         // (lib.optionalAttrs (attrs ? name || (attrs ? pname && attrs ? version)) {
          203|           name =
             |           ^
          204|             let

       … while evaluating the attribute 'out.outPath'

       at /home/siraben/.cache/nixpkgs-review/pr-131585/nixpkgs/lib/customisation.nix:156:13:

          155|             drvPath = assert condition; drv.${outputName}.drvPath;
          156|             outPath = assert condition; drv.${outputName}.outPath;
             |             ^
          157|           };

       … while querying the derivation named 'fedpkg-1.29'
```
